packages/expo-calendar/ios/EXCalendar/EXCalendar.m

```diff
@@ -330,10 +330,15 @@ - (EKEventStore *)eventStore
     calendarEvent.notes = nil;
   }
 
-  if (timeZone) {
-    calendarEvent.timeZone = [NSTimeZone timeZoneWithName:timeZone];
-  } else if (details[@"timeZone"] == [NSNull null]) {
+  if (details[@"timeZone"] == [NSNull null]) {
```

Comment:
Does this have to be explicit `[NSNull null]`? Couldn't it be `if (!details[@"timeZone"])`? It would catch both `null` and `undefined` from JS.

Reply:
Yes, this is fine because `null` and `undefined` have different meanings here. This method is used for both, creating and updating an event so when passed `timeZone` is `undefined` we want to do nothing, but if it's `null` then we want to reset it to `nil` 😉